File: TrainersTrainee/TrainersTrainee/Trainee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TrainersTrainee
{
    class Trainee
    {
        public static int count = 0;
        public string TraineeId { get; set; }
        public string TraineeName { get; set; }

        List<Training> trainings = new List<Training>();

        public Trainee()
        {
            count++;
        }
        
       
        
      
    }
}

File: UI/Front-End/OO with JS, TS DOM/TypeScript/LAb1/Trainee.ts
import { Training } from "./Training";

export class Trainee {
    TraineeName: string;
    trainings:Training[] = [];

    getTraineeName(): string {
        return this.TraineeName;
    }

    setTraineeName(_name: string) {
        this.TraineeName = _name;
    }

    getTrainings():Training[]{
        return this.trainings;
    }

    setTraining(_training : Training){
        this.trainings.push(_training);
    }
}